ci/qunit-selenium-runner.rb

```diff
@@ -1,22 +1,81 @@
 # frozen_string_literal: true
 
-require "qunit/selenium/test_runner"
+require "webdrivers"
 
-if ARGV[1]
-  driver = ::Selenium::WebDriver.for(:remote, url: ARGV[1], desired_capabilities: :chrome)
-else
-  require "webdrivers"
+# This class based on https://github.com/smontanari/qunit-selenium, with a few tweaks to make it easier to read output.
+# Copyright (c) 2014 Silvio Montanari
+# License: MIT
```

Comment:
@rafaelfranca is this okay, or do we need the full text?

Reply:
This is the most similar scenario to this that I could find: https://github.com/rails/rails/blob/main/guides/rails_guides/levenshtein.rb#L6

Reply:
reading the license I believe it needs to be the full text.

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.